helm_deploy/hmpps-incentives-api/values.yaml

```diff
@@ -2,6 +2,8 @@
 generic-service:
   nameOverride: hmpps-incentives-api
 
+  serviceAccountName: hmpps-incentives-api
```

Comment:
Created with
- https://github.com/ministryofjustice/cloud-platform-environments/pull/13799
- https://github.com/ministryofjustice/cloud-platform-environments/pull/13801
- https://github.com/ministryofjustice/cloud-platform-environments/pull/13802

As the IRSA module v2 service names only need to be unique within a namespace, rather than the cluster